dbt/include/bigquery/macros/adapters.sql

```diff
@@ -100,6 +100,9 @@
   {% if for_columns and config.persist_column_docs() and model.columns %}
     {% do alter_column_comment(relation, model.columns) %}
   {% endif %}
+  {% if config.persist_relation_docs() and 'description' in model %}
```

Comment:
```suggestion
  {% if config.persist_relation_docs() and model.description %}
```
can we do this instead of `in`?